src/button-set.jsx

```diff
@@ -0,0 +1,57 @@
+// @flow
+import React, {PropTypes as t} from 'react';
+
+// Types
+type ButtonKindType =
+  | 'danger'
```

Comment:
yeah me too!  I struggled with getting that removed for a long time but prettier does that automatically.